pkg/flag/vm_flags.go

```diff
@@ -0,0 +1,38 @@
+package flag
```

Comment:
Could you use AWSFlagGroup instead?
https://github.com/aquasecurity/trivy/blob/f369bd3e3dbc3f64135e361bca8059ecd4113e91/pkg/flag/aws_flags.go#L36-L42